### PR TITLE
Added change password

### PR DESCRIPTION
**Pull request by dctoon**
Sorry for the formatting.

Tests are not completed, need integration test environment information.

### CODE REVIEW

File: lib/users.js [hunk 1]
```diff
@@ -116,23 +117,118 @@ Users.prototype.validate = function (userName, password, callback) {
...
+    if (password === '') {
+      return callback(new WrongPassword(profile));
+    }
+
+    log('Change password for DN "' + profile.dn.green + '"');
+
+    var change = new ldap.Change({
+      operation: 'replace',
+      modification: {
+        userPassword: password
```
Comment (CriGoT) [inline lib/users.js]: Although `userPassword` is used by most LDAP directories we should move the attribute name to a configuration setting to allow it to be changed.

Comment (dctoon) [reply lib/users.js]: Added the parameter `CHANGE_PASSWORD_ATTRIBUTE_NAME`, defaults to `userPassword `.

File: test/users.tests.js [hunk 2]
```diff
@@ -125,4 +146,31 @@ describe('users', function () {
       expect(error.username).to.include('uqweu');
     });
   });
-});
+
+  describe.skip('changePassword with username and password', function () {
+    var profile;
+
+    before(function (done) {
+      users.changePassword('john', password, function (err, p) {
```
Comment (CriGoT) [inline test/users.tests.js]: `john` is the user we use to run all tests. Let's change this to another user we will create in AD to test this specifically. I'll provide the user details for the expectations

Comment (dctoon) [reply test/users.tests.js]: 👍

**Pull request merged**